Mutagen.Bethesda.Analyzers.Skyrim/Contextual/UnownedBedAnalyzer.cs

```diff
@@ -24,13 +24,15 @@ public void Analyze(ContextualAnalyzerParams param)
 
             foreach (var placedObject in cell.GetAllPlaced(param.LinkCache).OfType<IPlacedObjectGetter>())
             {
+                if (placedObject.IsDeleted) continue;
+
                 // Owned beds are not a problem
                 if (!placedObject.Owner.IsNull) continue;
 
                 if (!param.LinkCache.TryResolve<IFurnitureGetter>(placedObject.Base.FormKey, out var furniture)) continue;
                 if (!furniture.IsBed()) continue;
 
-                var context = param.LinkCache.ResolveSimpleContext(placedObject);
+                var context = param.LinkCache.ResolveSimpleContext<IPlacedObjectGetter>(placedObject.FormKey);
```

Comment:
Hmm, I can probably add another overload in Mutagen to avoid needing to do this.  Should be smart enough to identify that `placedObject` is `IPlacedObjectGetter` and use that as the type

Reply:
I've got a lot of bad experienced with not adding explicit getter types into resolve calls because they end up trying to look for the setter variants (which don't exist in the binary overlay) too many times. Might be that I'm getting too paranoid with this, just has been my experience to be better safe than sorry with this 😅

Reply:
Yeah, for sure.   Adding it explicitly is never bad if you're used to adding it